css/main.css

```diff
@@ -0,0 +1,433 @@
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+/* =====contact me section====  */
+
+.contact {
+  background-color: #fff;
+  padding-top: 80px;
+  min-height: 779px;
+}
+
+.contact-container {
+  background-color: #fffdfc;
+  min-height: 779px;
+  border-top-left-radius: 150px;
+  object-fit: cover;
+}
+
+.contact-container h2 {
+  padding-top: 103px;
+  color: #ffff;
+  text-align: center;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.contact-P {
+  padding-top: 12px;
+  padding-left: 10px;
+  padding-right: 10px;
+  color: #172b4d;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+}
+
+/* form section */
+
+.contact-form {
+  height: 100%;
+  margin-top: 2rem;
+}
+
+.form-section {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+input {
+  height: 2.9rem;
+  width: 20rem;
+  border: 0;
+  border-bottom: 2px solid #dbd8d7;
+  padding-left: 1rem;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  text-align: left;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  outline: none;
+}
+
+textarea {
+  width: 20rem;
+  height: 13rem;
+  border-radius: 5px;
+  border-style: none;
+  background-color: #fbf8f7;
+  padding-top: 1rem;
+  padding-left: 1rem;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  outline: none;
+}
+
+.getInTouch {
+  margin-top: 1rem;
+  border: 2px solid#1c1a19;
+  border-radius: 8px;
+  color: #1c1a19;
+  text-align: center;
+  justify-content: center;
+  padding: 1rem;
+  width: 10rem;
+  background-color: #fff;
+  margin-left: 10rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
```

Comment:
- [ ] Your button should be centered inside your form container, like the design from Figma. 🎯🎯

To do that, you can change or remove the property <code>margin</code> set to the button, and set the layout used by the parent element to **flexbox**, in this case, the <code>contact-form</code> to this following snippet of code:
```sh
display: flex;
flex-direction: column;
align-items: center;
```
🤓🤓🤓